homeassistant/components/zwave_js/light.py

```diff
@@ -109,8 +111,13 @@ def __init__(
         self._supported_color_modes = set()
 
         # get additional (optional) values and set features
-        self._target_value = self.get_zwave_value("targetValue")
-        self._dimming_duration = self.get_zwave_value("duration")
+        self._target_value = self.get_zwave_value(
+            "targetValue", add_to_watched_value_ids=False
+        )
+        self._target_color = self.get_zwave_value(
```

Comment:
Why didn't we need this value before? Are you only using it to determine whether or not it supports transitions?

Reply:
Yes only to check for transition support.